AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Campaign/Edit.cshtml

```diff
@@ -187,14 +187,14 @@
         <div class="form-group">
             <label asp-for="PrimaryContactPhoneNumber" class="col-md-2 control-label"></label>
             <div class="col-md-10">
-                <input asp-for="PrimaryContactPhoneNumber" class="form-control" />
+                <input type="tel" asp-for="PrimaryContactPhoneNumber" class="form-control" />
                 <span asp-validation-for="PrimaryContactPhoneNumber" class="text-danger"></span>
             </div>
         </div>
         <div class="form-group">
             <label asp-for="PrimaryContactEmail" class="col-md-2 control-label"></label>
             <div class="col-md-10">
-                <input asp-for="PrimaryContactEmail" class="form-control" />
+                <input type="email" asp-for="PrimaryContactEmail" class="form-control" />

```

Comment:
PrimaryContactEmail is already tagged with the [EmailAddress] data annotation which should output `type="email"`. Specifying the type again here is redundant.